lambda/client.py

```diff
@@ -0,0 +1,53 @@
+import requests
+
+
+class Client:
```

Comment:
Let's call it something more descriptive:
```suggestion
class LlmApiClient:
```